### DISCUSSION TITLE
奇怪的大规模“吸血”现象

### DISCUSSION THREAD

**Discussion by pullingtab** [PBH-BTN/PeerBanHelper#100] [讨论]
我注意到一件事。
似乎从23年的某个时间开始，出现了一些国内的吸血IP。
他们的特点是：
1. 使用相同的IP段，能在大量IP地之间切换，IP地址往往来自于内蒙古，辽宁，山东
2. 使用相同的端口
3. 反复下载少数几个文件
4. 这些IP会使用这样的peer ID：空ID，hp/*，dt/*，BitComet*，等

这群人有点奇怪。如果是单纯为了吸血，没道理总是吸同一个文件呀。
事实上，他们似乎根本不是在吸血，如果你真的连上他们，他们会给你发数据：
![奇怪的BT客户端](https://github.com/PBH-BTN/PeerBanHelper/assets/8113380/b0056a1d-e0e1-426d-a6d7-a3b0cd1d3fa5)
而且看上去，他们拥有各种名字。
客观上，他们的行为有利于提高这些资源的下载速度。
当然这是在你没有下完这个文件的情况下。如果你有了他们感兴趣的文件，并且好心的刻意做种，你的设备就会被大量的客户端连上，挤满你的上传速度。
![我下完了就开始吸我](https://github.com/PBH-BTN/PeerBanHelper/assets/8113380/61e084ae-11d2-4705-86b5-b29da8a93b78)
这种行为不能称为恶意的吸血，客观上他们只是在浪费你的上传带宽。他们不仅没有危害P2P网络的价值，相反还让某些文件的下载更快了。

有懂行的么？他们这是在干啥？带宽不要钱的么？
而且从他们的行为来看，是不是没有必要完全封禁他们？我们只需要想办法设置一个规则，给这些客户端在做种期间狂吸我们的客户端限个速，世界就会更美好。是这样么？

**Comment by Ghost-chu** [comment]
hp dt 不会给你发数据

BitComet * 是它们配置错误，所以发了

**Reply by pullingtab** [reply]
我去试了一下，hp确实不上传：
![image](https://github.com/PBH-BTN/PeerBanHelper/assets/8113380/171a09b6-6b08-4709-94c1-b7922c648003)

**Comment by SterileKiller** [comment]
我也发现了同样的情况，似乎是伪装成BitComet 2.0.4
